src/vellum/workflows/inputs/base.py

```diff
@@ -52,28 +73,15 @@ def __init__(self, **kwargs: Any) -> None:
                         code=WorkflowErrorCode.INVALID_INPUTS,
```

Comment:
[nit] could we add the `name` of the missing required input to this message?

[oos] we'll one day want to include all of the missing required inputs